javaagent-tooling/src/test/java/io/opentelemetry/javaagent/tooling/GuavaBoundedCacheTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+
+class GuavaBoundedCacheTest {
+
+  void testCache() {
+    fail("OH MY");
```

Comment:
Hmm, to be removed?

Reply:
Build is green! :D

Reply:
😱

Reply:
Yeah, those java unit tests in that dir don't get run.  🤷 
I'll convert it to groovy for now.